### PR TITLE
[DX-1507] Gateway 5.3.3 release notes 

### PR DESCRIPTION
**Pull request by Eopayemi**
### For internal users - Please add a Jira DX PR ticket to the subject!
[DX-1507](https://tyktech.atlassian.net/browse/DX-1507)

---

### Preview Link

[Preview](https://deploy-preview-5057--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/version-5.3)

### Description

5.3.3 release notes for gateway

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

[DX-1507]: https://tyktech.atlassian.net/browse/DX-1507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

___

### **PR Type**
Documentation, Enhancement

___

### **Description**
- Added comprehensive release notes for Tyk Gateway version 5.3.3.
- Included detailed sections for breaking changes, deprecations, upgrade instructions, release highlights, dependencies, downloads, and changelog.
- Updated tags to reflect the new version 5.3.3.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Added release notes for Tyk Gateway version 5.3.3</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md

<li>Added release notes for Tyk Gateway version 5.3.3.<br> <li> Included sections for breaking changes, deprecations, upgrade <br>instructions, release highlights, dependencies, downloads, and <br>changelog.<br> <li> Updated tags to include version 5.3.3.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5057/files#diff-ead598197a92e18014f4f7c35c341621948fb6cdf66d45c1dd05de21bc2b35aa">+135/-1</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 1]
```diff
@@ -19,6 +19,112 @@ tags: ["Tyk Gateway", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3", "changel
...
+
+
+### Changelog {#Changelog-v5.3.3}
+
+
+<!-- Required. The change log should include the following ordered set of sections below that briefly summarise the features, updates and fixed issues of the release.
+Here it is important to explain the benefit of each changelog item. As mentioned by James in a previous Slack message (https://tyktech.slack.com/archives/C044R3ZTN6L/p1686812207060839?thread_ts=1686762128.651249&cid=C044R3ZTN6L):
+"...it is important to document the customer impact for the work delivered, so we can share it with prospects/install base. For example:
+"New Chart delivers x and y benefit to a and b customer use cases. The business impact for them will be this and that" -->
+
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md:103]: @Eopayemi Can you add changelog sections for add and changed?

Comment (Eopayemi) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md:103]: Added in.

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 2]
```diff
@@ -19,6 +19,133 @@ tags: ["Tyk Gateway", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3", "changel
...
+<!-- This section should be a bullet point list that describes the issues fixed in the release. For each fixed issue explain:
+- What problem the issue caused
+- How was the issue fixed
+- Link to (new) documentation created as a result of a fix. For example, a new configuration parameter may have been introduced and documented for the fix
+- For OSS - Link to the corresponding issue if possible on GitHub to allow the users to see further info.
+Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->
+<ul>
+<li>
+ <details>
+ <summary>Optimised Key Creation to Reduce Redundant Redis Commands</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: Only Optimised and Redis should have capitals

```suggestion
 <summary>Optimised key creation to reduce redundant Redis commands</summary>
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 3]
```diff
@@ -19,6 +19,133 @@ tags: ["Tyk Gateway", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3", "changel
...
+
+Addressed an issue where creating or resetting a key caused an exponential number of Redis DeleteRawKey commands. Previously, the key creation sequence repeated for every API in the access list, leading to excessive deletion events, especially problematic for access lists with over 100 entries. Now, the key creation sequence executes only once, and redundant deletion of non-existent keys in Redis has been eliminated, significantly improving performance and stability for larger access lists.
+</details>
+</li>
+<li>
+<details>
+<summary>Resolved SSE Streaming Issue</summary>
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: Only Resolved and SSE should have capitals

```suggestion
<summary>Resolved SSE streaming issue</summary>
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @Eopayemi PR LGTM, as you have mentioned, PR needs updating with release date once available. I have made minor observations/suggestions on the PR relating to captilisation of changelog items in fixes section

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md [hunk 4]
```diff
@@ -19,6 +19,133 @@ tags: ["Tyk Gateway", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3", "changel
 <!-- Required. replace X.Y with this release and set the correct quarter of the year -->
 Our minor releases are supported until our next minor comes out.
 
+---
+## 5.3.3 Release Notes
+
+
+### Release Date xxx
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md]: Remember to add release date

**Review by dcs3spp — APPROVED** [review]
Thanks @Eopayemi I have approved and once release date added and PM approved will merge and release

**Review by lghiur — APPROVED** [review]

**Pull request merged**